docs/Building-Transmission.md

```diff
@@ -120,24 +120,34 @@ $ sudo make install
 ```
 
 ## On Windows ##
-For Windows XP and above there are several choices:
 
-### Cygwin environment ###
-With Cygwin https://cygwin.com/ installed, the CLI tools (transmission-remote, transmission-cli, etc.) and the daemon can be built easily.
+## Building transmission-daemon
+You need the following installed:
 
-No patches needed(\*), all the recent versions of Transmission build almost out-of-the-box (you need to install the prerequisites), and the CLI tools work better under Cygwin than those built with MinGW.
+* Visual Studio 2019 or greater (the Community Edition is sufficient - just make sure its C++ compiler, MSVC, is installed)
+* [ActivePerl](https://www.activestate.com/products/activeperl/) or [StrawberryPerl](https://strawberryperl.com)
+* [CMake](https://cmake.org/download/) (choose to add CMake to your path)
+* Possibly [Git for Windows](https://git-scm.com/download/win)
```

Comment:
You don't "possibly" need git, you definitely need it. However other build instructions omit git as a prerequisite because it is implied. So I would remove it here